chat-commands.js

```diff
@@ -2479,6 +2479,62 @@ exports.commands = {
 	},
 	blacklisthelp: [`/blacklist [username], [reason] - Blacklists the user from the room you are in for a year. Requires: # & ~`],
 
+	bb: 'battleban',
```

Comment:
I don't know how I feel about this alias. Does this really need an alias?

Reply:
Doesn’t matter to me but why not?

Reply:
Because it's an unclear alias. I think this command isn't going to be used often enough to warrant it, and its not hard to type out ``/battleban``. Think of how ``/b`` was removed to global ban, because people thought it might do something else. Same idea here.

Reply:
Yeah I already removed it, although I don’t think it’s “unclear.” It’s definitely not going to be used enough to warrant an alias tho. Also “b” was removed from globalban because of the serverity of the punishment and because it was changed over to an alias for /roomban. If this had more use I would definitely argue /bb is fine.